interfaces/portal/src/app/components/query-table/query-table.component.html

```diff
@@ -219,7 +236,17 @@
           } @else {
             @let cellText = getCellText(column, item);
             @if (column.type === 'multiselect') {
-              @if (column.getCellChipData) {
+              @if (column.displayAsPlainText) {
```

Comment:
"plain text"? But then with fancy icons. :D

Reply:
Let me rethink this a second and get back to you

Reply:
🍿

Reply:
[`cd504b2` (#7188)](https://github.com/global-121/121-platform/pull/7188/commits/cd504b29b2f3f88abc776937bd853a2494db0363)